### ISSUE TITLE
1.3.4 release

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#1832]
I could do a release this week. Would you want to wait until https://github.com/FlowCrypt/flowcrypt-android/issues/1829 is done?

**Comment by DenBond7** [comment]
Nope, I'm moving #1829(refactoring and fixing deprecated code) to the next milestone. I'll notify you when 1.3.4 will be ready.

**Comment by DenBond7** [comment]
I've reviewed ACRA logs and found one issue https://github.com/FlowCrypt/flowcrypt-android/issues/1834 that I'd like to fix in the current milestone. Can we wait until Thursday for a new release?

**Comment by DenBond7** [comment]
@tomholub  #1834 is ready to be merged. After that, we can make a new release.

**Comment by tomholub** [comment]
- [ ] write customer
 - [ ] build release
 - [ ] release consumer 10%
 - [ ] release enterprise

**Comment by tomholub** [comment]
I'll be releasing consumer releases with 20% rollout if that's ok with you. The builds have been quite stable lately :-)

**Comment by DenBond7** [comment]
`release enterprise 2022-06-23` - done!
